### PR TITLE
Upgrade phoenix to 1.5.0 and add live view dashboard

### PR DESCRIPTION
**Pull request by josepjaume**
## ✍️ Description

This updates Phoenix to the 1.5.0-rc.1 version, and adds [live_view_dashboard](https://github.com/phoenixframework/phoenix_live_dashboard).

Additionally, it adds support for:

* Request logging, which shows logs for a particular request (super cool!)
* Metrics, adding support for telemetry to monitor the system.

## ⚠️ IMPORTANT

Before merging this PR, an `ADMIN_PASSWORD` environment variable needs to be added to each deployment (staging and production).

## ✅ Fixes

*None*

## 🕵️‍♀️ QA

Before requesting a review, please make sure that:

- [x] Preview is working on Netlify, Heroku or similar.
- [x] Manually check that it's working.
- [x] Add documentation and tests if needed.

## 📸 Screenshots

image

## 🔗 Relevant URLs

* https://regalocal-upgrade-phoen-frr9ej.herokuapp.com/dashboard/ - User "admin", password "admin1234" (it's parameterized as a secret on production).

## 📕Extra Documentation

* [Phoenix 1.5 changelog](https://github.com/phoenixframework/phoenix/blob/master/CHANGELOG.md)
* [Phoenix Live Dashboard](https://github.com/phoenixframework/phoenix_live_dashboard)

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -10,12 +10,15 @@ use Mix.Config
 config :regalocal,
   ecto_repos: [Regalocal.Repo]
 
+config :regalocal, :basic_auth, username: "admin", password: "admin1234"
```
Comment (mrcasals) [inline config/config.exs:13]: I think this sould be fetched from an environment variable, right?

Comment (josepjaume) [reply config/config.exs:13]: It is on production! Check it out:

https://github.com/codegram/regalocal/pull/94/files#diff-4c347b97e9c4e8c027f508a09c804eabR37

Comment (mrcasals) [reply config/config.exs:13]: Ah, gotcha! I didn't understand the `releases.exs` file, thanks!

**Comment by josepjaume** [comment]
This releases thing is a neat elixir feature:
https://elixir-lang.org/getting-started/mix-otp/config-and-releases.html

**Pull request merged**